modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -636,7 +610,7 @@ FeatureFloodCount::updateFieldInfo()
 
       // Loop over the ghosted ids to update cells with ghost information
       for (auto entity : feature._ghosted_ids)
-        _ghosted_entity_ids[entity] = feature_number;
+        _ghosted_entity_ids[entity] = 1;
```

Comment:
Yeah, I'm sure...

I changed this line because I really don't want to see a "multi-colored" field when I look at ghosted ids. Using processor_id might be interesting but certainly not something related to the feature or grain. Just using a constant gives the "expected" looking field.